Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift

```diff
@@ -207,18 +216,32 @@ class AssignmentListViewController: UIViewController, ColoredNavViewProtocol, Er
     }
 
     @objc func refresh(_ control: UIRefreshControl) {
-        fetchData()
+        fetchData(showActivityIndicator: true)
     }
 
     func showSpinner(show: Bool = true) {
         if show {
             tableRefresher.beginRefreshing()
-        } else {
+        } else if tableRefresher.isRefreshing {
             tableRefresher.endRefreshing()
```

Comment:
Calling `endRefreshing` was what was causing the scrolling to be choppy while paging.  Don't call this when paging loads are occurring.